includes/RestApi/RestApiFilter.php

```diff
@@ -131,7 +131,11 @@ public static function header_menu_rename_pages( $response, array $handler, \WP_
 	 * @return boolean
 	 */
 	public static function is_request_from_onboarding_flow( \WP_REST_Request $request ) {
-		return false !== stripos( $request->get_header( 'referer' ), 'page=nfd-onboarding' );
+		if ( ! $request->get_header( 'referer' ) ) {
+			return false;
+		} else {
+			return false !== stripos( $request->get_header( 'referer' ), 'page=nfd-onboarding' );
+		}
```

Comment:
```suggestion
		$referrer = $request->get_header( 'referer' );
		if ( ! $referrer ) {
			return false;
		}
		return false !== stripos( $referrer, 'page=' . WP_Admin::$slug );
```
We'll also have to import `NewfoldLabs\WP\Module\Onboarding\WP_Admin`

Reply:
have made the changes according to the comment